integration-tests/mockApis/auth.js

```diff
@@ -616,7 +616,13 @@ const verifyAuthCreateUser = () =>
 
 module.exports = {
   getSignInUrl,
-  stubSignIn: (username, roles, tokenRoles) =>
+  stubSignIn: (username, roles) => {
+    let tokenRoles = []
+    roles.forEach((role) => tokenRoles.push(`ROLE_${role.roleCode}`))
```

Comment:
The call to stubSignIn now only takes 2 params so any calls to this should be updated to not pass in tokenRoles - think there are only 4 files with this issue.

Reply:
Thanks Karen - addressed now